bin/asciibinder

```diff
@@ -104,6 +115,23 @@ Description:
   Creates a new, bare AsciiBinder repo in the specified directory.
 EOF
     end
+  when "clone"
+    Trollop::options do
+      banner <<-EOF
+Usage:
+  #$0 clone <options> <git_repo_url>
+
+Description:
+  Clones an existing AsciiBinder repo to the current directory.
+  Under the default behavior, AsciiBinder will attempt to set up
+  tracking branches based on the contents of _distro_map.yml,
+  but this can be suppressed (see Options).
+
+Options:
+EOF
+      opt :nobranches, "Don't attempt to create tracking branches after cloning.", :type => :bool, :default => false
+      opt :targetdir, "Specify the pathname of the local directory for cloning.", :default => ''
```

Comment:
I think simpler options would be nicer. `:dir` or `:directory` is simpler.  Either way, if you go with target dir, it might be nicer as `:target_dir` (which would create an option called `--target-dir`